dscreator/main.py

```diff
@@ -131,5 +132,36 @@ def sios(max_time_slice: int = 24, stop_after_n_files: int = -1, acdd: bool = Fa
     runner.start()
 
 
+@app.command()
+def nrt_color_fantasy(max_time_slice: int = 24, stop_after_n_files: int = -1, acdd: bool = False):
+    """Build nrt color fantasy dataset from data in tsb"""
+
+    logging.info("Exporting NRT FA dataset")
+    trajectory_extractor = ferrybox.extractor.TrajectoryExtractor(
+        create_engine(SETTINGS.tsb_connection_str),
+        variable_codes=["temperature", "salinity", "oxygen", "chlorophyll", "turbidity", "cdom"],
+        variable_uuid_map=ferrybox.uuid_variable_code_mapper.MAPPER["FA_20"],
+        qc_flags=[1],
+    )
+
+    dataset_builder = trajectories.ferrybox.DailyFantasy(
+        uuid="no.niva:af11ba01-dfe3-4432-b9d2-4e6fd10714db",
+        dataset_name="nrt_color_fantasy",
+        station_name="color_fantasy",
+        project_name="NorSoop",
```

Comment:
I guess now the project should be AquaInfra? NorSOOP ended a couple of years ago

Reply:
this is just used to specify the storage location, but guess we can update it:)